sqlite.go

```diff
@@ -8,6 +8,24 @@
 // For details see https://sqlite.org/c3ref/open.html#urifilenames.
 //
 //
+// Initializing connections or tracing
+//
+// The database/sql package creates connections unexpectedly, so if you
```

Comment:
I'm not sure this is quite the right messaging. :P

I'd cut the first part entirely.

```go
// If you want to do initial [...]
```